openshift/gabi-no-cluster-resources.template.yaml

```diff
@@ -57,6 +75,24 @@ objects:
             name: gabi-tls
         - image: quay.io/app-sre/gabi:${IMAGE_TAG}
           name: gabi
+          livenessProbe:
+            exec:
```

Comment:
why not `httpGet`?

Reply:
The port 8080 is not exposed. I think it's required to use httpGet probes

Reply:
Ok, that wasn't true. I've tried it because I've seen some examples without exposing the port and it works. 

I changed the probes to httpget.